lib/l10n/en.dart

```diff
@@ -261,5 +261,10 @@ class English extends AppLocalizations {
   String get tunerTutorialAdjust => 'Tap here to adjust the concert pitch or play a reference tone.';
   String get tunerTutorialStartStop => 'Tap here to start and stop the tuner.';
 
+  String formatNumber(double num) => NumberFormat.decimalPattern('en').format(num);
+
+  String formatNumberAndRoundToOneDecimal(double num) =>
+      NumberFormat.decimalPattern('en').format(double.parse(num.toStringAsFixed(1)));
```

Comment:
```suggestion
      NumberFormat.decimalPattern(locale).format(double.parse(num.toStringAsFixed(1)));
```

Use getter for locale inside this class.